qml/EventUtils.qml

```diff
@@ -78,6 +78,18 @@ QtObject{
             // E.g. "Daily; until 12/12/2014."
             str = i18n.tr("%1; until %2").arg(recurrence).arg(rule.limit.toLocaleString(Qt.locale(), dateFormat))
         }
+	
+	if (rule.interval !== undefined) {
+            if (rule.interval == 2) {
+                str = str + i18n.tr(", every 2nd time");
```

Comment:
This creates a problem in translation: These are now 3 strings in English, but might be 2, 4 or more strings in other languages. And its m:n possibly, so you cannot make assumptions about how many strings will be needed. Lets talk about it, its a perfect case for what makes good software better than "average" software ;)

Reply:
Please see how it's done a few lines above:
```
   // TRANSLATORS: the argument refers to multiple recurrence of event with count .
   // E.g. "Daily; 5 times."
   str = i18n.tr("%1; %2 time", "%1; %2 times", rule.limit).arg(recurrence).arg(rule.limit)
```
See also https://docs.ubuntu.com/phone/en/apps/api-qml-current/Ubuntu.Components.i18n and especially https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

Reply:
Well a traditional singular/plural form will not be sufficient here, its not a plural but first, second, third are cardinal words? I dont even know the term. The example above is the simple case. I would like to see if there is anything written about this more complex situation.

Reply:
I don't have a better idea, than doing the same up to maybe 7 for example. I think its extremely unlikely having an event occuring every 7th week/month/year or something like that... But if you have better ideas, just bring them up... :)

Reply:
Okay, they are called ordinal numbers. And what to do with it...?
PHP example: https://stackoverflow.com/questions/2453594/localize-ordinal-numbers
But Qt: https://bugreports.qt.io/browse/QTBUG-55494
How to continue with that?

Reply:
Should we rework the sentence to work with cardinal numbers instead?
For example, instead of using "2nd time", avoid using the word "time" at all, and use a more explicit form like "every 2 days": this would allow as to use cardinal numbers and therefore use the standard way of translating these sentences.
This of course has the problem that we need a few more strings (depending on whether the period is day, week, month, etc.) but I think it's worth it.

Reply:
@mardy Okay, I've done, what you suggested :) please review again.